.github/workflows/node-ci.yml

```diff
@@ -74,9 +74,12 @@ jobs:
     strategy:
       matrix:
         os:
-          - [self-hosted, macOS, ARM64]
+          - macos-12
           - ubuntu-20.04
+        node: [ 10, 14, 16, 18 ]
```

Comment:
Let's focus on the LTS here. Believe node 10 had EOL 30 Apr 2021, so it's not safe to use.

Reply:
Node 10 was kept because it was the last node version mapbox-gl-native was built for. It made switchover/testing/comparison easier if you could have a release that supports both mapbox and maplibre since there aren't a lot of examples out there.

Reply:
I'm fine with removing it here, but it was really useful for bug comparison, like the app I made for to compare the label clipping issue ( https://github.com/acalcutt/maplibre-label-clipping-test ), which I could switch between mapbox and maplibre and see the difference in results.

Reply:
Okay, if it's useful to have node 10, and it doesn't add complexity to the codebase moving forward, then let's keep it for now.

Reply:
Just saw this comment. i added node 10 back for now. I agree we can remove it soon, but having at least a few releases with it would be good for migrations.